UnitTests/TestCategoryDisplayService.php

```diff
@@ -4,7 +4,6 @@
 require_once '../src/Entities/CategoryEntity.php';
 
 use PHPUnit\Framework\TestCase;
-
```

Comment:
You've gone kind of ballistic on removing whitespace. Some whitespace - a line - between meaningfully different 'stuff' is actually very helpful when reading code.

Generally: 1 line between functions/methods, before a class declaration